src/main/java/org/spongepowered/api/text/Texts.java

```diff
@@ -175,25 +203,25 @@ public static Text of() {
      * @return The built text object
      */
     public static Text of(Object... objects) {
+        // Shortcut for lonely TextRepresentables
+        if (objects.length == 1 && objects[0] instanceof TextRepresentable) {
```

Comment:
Why would you try to apply some custom formatting to a TextRepresentable, without actually passing the formating to this method?
If there is formatting passed then objects.length!=1.

If you pas the formatting to this method then you will enter line 236 and get your formatting merged into that TextRepresentable.
With the old logic you couldn't even change the color of a Text argument that has been passed to this method:
`Texts.of("Hello " , TextColors.RED, somePreExistingTextObject)` would end up with RED being ignore.